interfaces/acados_matlab_octave/acados_ocp.m

```diff
@@ -39,6 +39,7 @@
         model_struct
         opts_struct
         acados_ocp_nlp_json
+        ext_fun_type
```

Comment:
I'm not 100% convinced about adding this alone here, but for now it's fine, we can always change later on